node/src/server-modules/GlideJson.ts

```diff
@@ -195,6 +195,42 @@ export class GlideJson {
         return _executeCommand(client, args, options);
     }
 
+    /**
+     * Retrieves the JSON values at the specified `path` stored at multiple `keys`.
+     *
+     * @remarks When in cluster mode, all `keys` must map to the same hash slot.
+     *
+     * @param client - The client to execute the command.
+     * @param keys - The keys of the JSON documents.
+     * @param path - The path within the JSON documents.
+     * @param options - (Optional) See {@link DecoderOption}.
+     * @returns
+     * - For JSONPath (path starts with `$`):
+     *       Returns a stringified JSON list replies for every possible path, or a string representation
+     *       of an empty array, if path doesn't exist.
+     * - For legacy path (path doesn't start with `$`):
+     *       Returns a string representation of the value in `path`. If `path` doesn't exist,
+     *       the corresponding array element will be `null`.
+     * - If a `key` doesn't exist, the corresponding array element will be `null`.
```

Comment:
What if `keys` is empty? It seems like at least one key is required.

Reply:
Command will interpret `path` and as a key and will complain that `path` is missing.

Reply:
I think that error is confusing because as a user of the API, I clearly gave a path. Would be nice if we could avoid this case somehow with a NonEmptyList type, but I think the best we could do here is just document it.

Reply:
Maybe @acarbonetto can weigh in here too, since he usually has opinions on these kinds of issues.

Reply:
There is no `number_of_keys` argument in the command line, which makes this API more vulnerable to mistakes. See `BZMPOP` for example.
If you forgot `path` argument, command will interpret last key as a path and will return no errors.

Reply:
@Yury-Fridlyand we could add a condition into the command that if `[]` is passed (empty list) it returns a requestError right away. Then we won't get a `key doesn't exist` error - which is confusing.

Reply:
Apparently, this returns a wrong number of arguments error in actuality, so this isn't an issue after all.